ci-operator/step-registry/agent/post/agent-post-chain.yaml

```diff
@@ -3,7 +3,7 @@ chain:
   steps:
   - ref: baremetalds-devscripts-gather
   - ref: agent-gather
-#  - chain: gather
+  - chain: gather
```

Comment:
At this point I guess it could be better to ref directly the baremetalds-ipi-post chain, after the agent-gather

Reply:
That causes duplicate name error as seen [here](https://github.com/openshift/release/pull/40768#issuecomment-1621748935)